i18n/locales/es-MX/hourofcode/how-to/events.md

```diff
@@ -10,7 +10,7 @@ nav: how_to_nav
 ## 1. Prepárate para tu evento
 
 - Elige lugar, fecha y hora.
-- Envía [una carta](https://docs.google.com/a/code.org/document/d/1eP41sKW7y0qq_JvkRIgZK8dWYICaGRZ4CCDETXa78wY/edit) invitando al alcalde, diputado, presidente autonómico o empresario influyente para que de una charla. Consulta nuestras [herramientas prácticas](%=resolve_url('/files/elected-official.pdf')%) cuando alojes a un funcionario durante una hora de código para obtener más información.
+- Send [a letter](https://hourofcode.com/promote/resources#sample-emails) to invite your local mayor, congressman, governor, or influential business person to speak. Check out our [how-to toolkit](%=localized_file('/files/elected-official.pdf')%) when hosting an elected official during an Hour of Code for more info.
```

Comment:
Did we lose the translation because we changed from `resolve_url` to `localized_file`?

Reply:
Yep, see https://github.com/code-dot-org/code-dot-org/pull/21473#discussion_r177273273